src/global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "5.0.100",
+    "version": "6.0.x",
```

Comment:
In the past, this was not working and required an explicit version (not like in global.json). Are you sure that it doesn't fetch the latest?

Reply:
I'm not aware of this issue. Do you have a link to check if this is fixed already?
Alternatively I can change this to an explicit version, if you want.

Reply:
I don't think it is fixed as the [documentation](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x#version) still states that you need an actual version and wildcards (e.g even `x`) are not supported

So just to be safe, easier to put an actual version. I pushed the suggestion.